jobsdb/internal/constants/constants.go

```diff
@@ -0,0 +1,9 @@
+package constants
+
+const (
+	TableName   = "tableName"
+	WorkspaceID = "workspaceID"
+	Err         = "error"
+	CustomVal   = "customVal"
+	TablePrefix = "tablePrefix"
+)
```

Comment:
Don't we need project-wide logfield constants? Are we supposed to be redefining the same constants in other packages too?

Reply:
those were my thoughts too.
It's being worked in another PR(albeit not in a project-wide fashion), so this is kind of a placeholder for now.